compiler/src/Language/Bond/Codegen/Cpp/Types_h.hs

```diff
@@ -278,17 +278,20 @@ namespace std
                 getAllocator BT_MetaFullName =  [lt|.get_allocator()|]
                 getAllocator _ = mempty
 
-        -- copy constructor with allocator
-        allocatorCopyCtor alloc = [lt|
-        #{declName}(const #{declName}&#{otherParam}, const #{alloc}&#{allocParam})#{initList}#{ctorBody}
-        |]
+        -- copy/move constructor with allocator
+        allocatorCopyOrMoveCtor fieldDecl paramValue alloc = [lt|
```

Comment:
`fieldDecl` make me think this is handling each field one-by-one. Perhaps `paramDecl` or `otherParamDecl`? (If `otherParamDecl` is picked, then consider `otherParamValue` instead of `paramValue`.)

Reply:
Updated.